python/MooseDocs/common/parse_settings.py

```diff
@@ -25,14 +25,15 @@ def get_settings_as_dict(settings):
         output[key] = value[0]
     return output
 
-def match_settings(known, raw):
+def match_settings(defaults, raw):
     """
     Parses a raw string for key, value pairs separated by an equal sign.
 
     Inputs:
         default[dict]: The default values for the known keys.
         raw[str]: The raw string to parse and inject into a dict().
     """
+    known = dict(defaults)
     unknown = dict()
```

Comment:
@aeslaughter 
Wanted to point this out... this method was overriding the defaults when it returned the object back.

Reply:
Good catch.